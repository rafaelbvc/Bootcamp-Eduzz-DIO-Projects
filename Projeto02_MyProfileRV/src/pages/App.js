import React from 'react';
import BckGround from '../components/containers/BckGround';
import { SafeAreaView, View } from 'react-native';

const App = () => {
  return (
    <>
      <SafeAreaView>
        <BckGround />
      </SafeAreaView>
    </>
  ); 
};


export default App;