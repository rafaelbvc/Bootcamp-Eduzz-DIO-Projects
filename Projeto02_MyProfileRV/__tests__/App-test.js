import 'react-native';
import React from 'react';
import App from '../src/pages/App';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  renderer.create(<App />);
});

it('renders the profile image', () => {

  const profileImage = renderer.create('img');

  expect(profileImage).toMatchSnapshot();
});

it('renders texts at app', () => {

  const profileText = renderer.create('text');

  expect(profileText).toMatchSnapshot();
});

it('renders buttons at app', () => {

  const profileButtons = renderer.create('button');

  expect(profileButtons).toMatchSnapshot();
});

it('renders modals at app', () => {

  const profileModal = renderer.create('modal');

  expect(profileModal).toMatchSnapshot();

});
