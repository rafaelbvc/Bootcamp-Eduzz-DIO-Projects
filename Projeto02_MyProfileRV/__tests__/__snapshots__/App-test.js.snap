// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`randers texts at app 1`] = `"text"`;

exports[`renders buttons at app 1`] = `"button"`;

exports[`renders texts at app 1`] = `"text"`;

exports[`renders the profile image 1`] = `"img"`;
